bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/env/Asterism.scala

```diff
@@ -36,7 +36,7 @@ trait Asterism {
     * only possibilities.
     * TODO: should we Church encode this instead?
     */
-  def targets: Target \/ (SiderealTarget, SiderealTarget)
+  def targets: Target \/ (Target, Target)
```

Comment:
I'm not sure `isSidereal` makes sense for an asterism anyway.  Perhaps if it is implemented as "are all targets sidereal"?